src/server/game/Quests/QuestDef.h

```diff
@@ -388,6 +388,9 @@ class TC_GAME_API Quest
         uint32 _startItemCount        = 0;
         uint32 _rewardMailSenderEntry = 0;
         uint32 _specialFlags          = 0; // custom flags, not sniffed/WDB
+
+        // Helpers
+        uint32 RoundXPValue(uint32 xp) const;
```

Comment:
This helper function doesn't need access to any of the Quest object members, it can be made static, or even live only in the .cpp as non-member function